MDC2020/gen_Mix.sh

```diff
@@ -98,7 +99,7 @@ if [ $1 == "NoPrimary" ]; then
   --auxinput=1:physics.filters.NeutralsFlashMixer.fileNames:NeutralsFlashCat$3.txt
 else
   generate_fcl --dsconf="$2$4" --dsowner=mu2e --description="$1Mix" --embed template.fcl \
-  --inputs="$1$4.txt" --merge-factor=1 \
+  --inputs="$1$3.txt" --merge-factor=1 \
```

Comment:
I am not sure about this: the version of the primary will depend on the version used in the POMS mixing campaign, because there we create the primary files. E.g. the beam files can be MDC2020k and CeEndpoint can be MDC2020m.